website/docs/reference/resource-configs/grants.md

```diff
@@ -1,95 +1,67 @@
 ---
-resource_types: all
-datatype: "{<dictionary>}"
+resource_types: [models,seeds,snapshots]
+datatype: "string"
 default_value: {}
+id: "grants"
 ---
 
+<Snippet src="available-prerelease-beta-banner" />
+
+You can manage access to the datasets you're producing with dbt by using grants. To implement these permissions, define grants as resource configs on each model, seed, or snapshot. Define the default grants that apply to the entire project in your `dbt_project.yml`, and define model-specific grants within each model's SQL or YAML file.
+
+The grant resource configs enable you to apply permissions to a specific model and set of recipients as soon as the model finishes building.
```

Comment:
High-level message:

> As soon as your model, seed, or snapshot finishes building, dbt ensures that the grants on its view or table match **exactly** the `grants` you have configured.

> Behind the scenes, dbt will try to take the most efficient approach to updating grants. This approach will vary based on the adapter you're using, and whether dbt is replacing or updating an object that already exists. You can always check [the debug logs](https://docs.getdbt.com/reference/events-logging) for the full set of `grant` and `revoke` statements that dbt runs.

The language there is a bit clunky, but that's the big idea. As a user, you can trust in an exact match. dbt handles the nuance of the implementation.

Reply:
Added here: https://github.com/dbt-labs/docs.getdbt.com/pull/1642/commits/64fcd6d90666c933e0c25e4324adb0936dc2338b